.lando.yml

```diff
@@ -49,12 +49,24 @@ services:
     hogfrom:
       - appserver
 
+  node:
+    type: node
+    globals:
+      gulp: latest
+      gulp-cli: latest
+    build:
+      - npm i -g @frctl/fractal
+      - cd /app/fractal && yarn install
```

Comment:
We should use `npm` and not `yarn` (https://github.com/inviqa/engineering-guidelines/blob/master/platform/node.js/version.md#yarn).

I know the source is in the Node section, but it will be moved to the FE section soon enough when I get a chance.